frontend-react/src/components/Table/Table.tsx

```diff
@@ -115,69 +147,121 @@ const Table = ({ config, filterManager, cursorManager }: TableProps) => {
                         );
                     }
                 })}
+                {enableEditableRows ? (
+                    // This extends the header bottom border to cover this column
+                    <th key={"edit"}>{""}</th>
+                ) : null}
             </tr>
         );
     };
 
-    const showMappedValue = (
-        columnConfig: ColumnConfig,
-        object: TableRow
-    ): string => {
+    const showMappedValue = (columnConfig: ColumnConfig, object: TableRow) => {
         if (columnConfig.valueMap) {
             return (
-                columnConfig.valueMap?.get(object[columnConfig.dataAttr]) ||
-                object[columnConfig.dataAttr]
+                <span>
+                    {columnConfig.valueMap?.get(
+                        object[columnConfig.dataAttr]
+                    ) || object[columnConfig.dataAttr]}
+                </span>
             );
         } else {
-            return object[columnConfig.dataAttr];
+            return <span>{object[columnConfig.dataAttr]}</span>;
         }
     };
 
-    const renderRow = (object: TableRow, columnConfig: ColumnConfig) => {
-        let displayValue = object[columnConfig.dataAttr];
-        // Transforms value if transform function is given
+    const Column = ({
+        rowIndex,
+        colIndex,
+        rowData,
+        columnConfig,
+        editing,
+    }: ColumnProps) => {
+        const tableData = (child: ReactNode) => (
+            <td key={`${rowIndex}:${colIndex}`}>{child}</td>
+        );
+
         if (columnConfig.transform) {
-            displayValue = columnConfig.transform(displayValue);
+            rowData[columnConfig.dataAttr] = columnConfig.transform(
+                rowData[columnConfig.dataAttr]
+            );
         }
 
         if (columnConfig.link) {
             // Render column value as NavLink
-            return (
+            return tableData(
                 <NavLink
                     className="usa-link"
                     to={`${columnConfig?.linkBasePath || ""}${
-                        object[columnConfig?.linkAttr || columnConfig.dataAttr]
+                        rowData[columnConfig?.linkAttr || columnConfig.dataAttr]
                     }`}
                 >
                     {columnConfig.valueMap
-                        ? showMappedValue(columnConfig, object)
-                        : displayValue}
+                        ? showMappedValue(columnConfig, rowData)
+                        : rowData[columnConfig.dataAttr]}
                 </NavLink>
             );
         } else if (columnConfig.actionable) {
```

Comment:
It might be worth making this a `columnType` prop that can be used in a switch to return; keeps it cleaner and ready for expansion.

```typescript
interface ColumnConfig {
        ...,
        columnType: Linkable | Actionable 
}
```
the content of the column can be handled by a function to encapsulate the logic.

Do `Linkable` columns need to also be editable fields? 🤔 `Actionable` columns...?